library/std/src/os/fd/mod.rs

```diff
@@ -1,16 +1,25 @@
 //! Owned and borrowed Unix-like file descriptors.
+//!
+//! This module is supported on Unix platforms and WASI, which both use a
+//! similar file descriptor system for referencing OS resources.
 
 #![stable(feature = "io_safety", since = "1.63.0")]
```

Comment:
This should be `1.66.0`, right?

Reply:
Good catch; I've now filed https://github.com/rust-lang/rust/pull/106867 to fix this.